app/assets/stylesheets/plugins/wice_grid.scss

```diff
@@ -1,148 +1,81 @@
-.wg-detached-filter,
-.wice-grid {
+.wg-detached-filter, .wice-grid{
```

Comment:
Each selector in a comma sequence should be on its own single line<br>Opening curly brace `{` should be preceded by one space